docs/sphinx/_ext/dagster-sphinx/dagster_sphinx/__init__.py

```diff
@@ -155,7 +155,7 @@ def check_custom_errors(app: Sphinx, exc: Optional[Exception] = None) -> None:
     if len(dagster_errors) > 0:
         for error_msg in dagster_errors:
             logger.info(error_msg)
-        raise Exception(
+        logger.error(
```

Comment:
There were over 100 undocumented public APIs listed when this ran. Not sure why it wasn't being caught when building our JSON docs but they are being caught now, so had to make this log instead of Raise.